templates/federated_reporting/dump.sh

```diff
@@ -76,3 +76,13 @@ else
   mv "$in_progress_file" "$CFE_FR_TRANSPORT_DIR/$CFE_FR_FEEDER.sql.$CFE_FR_COMPRESSOR_EXT"
   chown "$CFE_FR_FEEDER_USERNAME" "$CFE_FR_TRANSPORT_DIR/$CFE_FR_FEEDER.sql.$CFE_FR_COMPRESSOR_EXT"
 fi
+
+if [ -n "CFE_FR_SUPERHUB_HOSTKEYS" ]; then
+  log "Copying for multiple superhubs: $CFE_FR_SUPERHUB_HOSTKEYS"
+  for superhub_hostkey in $CFE_FR_SUPERHUB_HOSTKEYS; do
+    mkdir -p "$CFE_FR_TRANSPORT_DIR/$superhub_hostkey"
+    cp "$CFE_FR_TRANSPORT_DIR/$CFE_FR_FEEDER.sql.$CFE_FR_COMPRESSOR_EXT" "$CFE_FR_TRANSPORT_DIR/$superhub_hostkey/"
```

Comment:
Do I understand correctly that this makes a copy of the data for each superhub?

So, say the dump is 10G, if you feed 2 superhubs, now you will need 20G of disk on the feeder?

If so, perhaps we make hard links?

Reply:
done. seems to work.